src/hotspot/share/gc/shenandoah/shenandoahCardTable.cpp

```diff
@@ -23,6 +23,7 @@
  */
 
 #include "precompiled.hpp"
+#include "gc/shared/memset_with_concurrent_readers.hpp"
```

Comment:
What's this?

Reply:
Not needed. Removing it.

(This is a remnant from a former version of mine where more functionality was hosted in this class. Tsktsk.)